templates/data-store-pvc.yaml

```diff
@@ -32,7 +32,7 @@
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: {{ $name }}-{{ $namespace }}
+  name: {{ $name }}
```

Comment:
does it not need to get deployed to a namespace?

Reply:
It is deployed into a namespace but this does not need to be included in the pvc name.